index.html

```diff
@@ -0,0 +1,198 @@
+              qui culpa quas voluptatem? Itaque, corrupti. Obcaecati.
+            </div>
+          </div>
+
+          <div class="card">
+            <div class="card-img">
+              <i class="fa fa-users fa-2x" aria-hidden="true"></i>
+            </div>
+            <div class="card-title">Language</div>
+            <div class="card-txt">
+              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Tenetur,
+              minima non doloribus harum corrupti dolores nihil illo voluptates,
+              qui culpa quas voluptatem? Itaque, corrupti. Obcaecati.
+            </div>
+          </div>
+
+          <div class="card">
+            <div class="card-img">
+              <i class="fa fa-mobile fa-2x" aria-hidden="true"></i>
+            </div>
+            <div class="card-title">Computer</div>
+            <div class="card-txt">
+              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Tenetur,
+              minima non doloribus harum corrupti dolores nihil illo voluptates,
+              qui culpa quas voluptatem? Itaque, corrupti. Obcaecati.
+            </div>
+          </div>
+
+          <div class="card">
+            <div class="card-img">
+              <i class="fa fa-users fa-2x" aria-hidden="true"></i>
+            </div>
+            <div class="card-title">Programming</div>
+            <div class="card-txt">
+              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Tenetur,
+              minima non doloribus harum corrupti dolores nihil illo voluptates,
+              qui culpa quas voluptatem? Itaque, corrupti. Obcaecati.
+            </div>
+          </div>
+
+          <div class="card">
+            <div class="card-img">
+              <i class="fa fa-mobile fa-2x" aria-hidden="true"></i>
+            </div>
+            <div class="card-title">Mobile</div>
+            <div class="card-txt">
+              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Tenetur,
+              minima non doloribus harum corrupti dolores nihil illo voluptates,
+              qui culpa quas voluptatem? Itaque, corrupti. Obcaecati.
+            </div>
+          </div>
+
+          <a href="#" class="service">See all services</a>
+        </div>
+        <div class="see-all-div">
+          <h3 class="see-all">Training Teams</h3>
+          <div class="trainer-container" id="trainer-container"></div>
+        </div>
+      </section>
+    </div>
+    <section>
+      <div class="container-part">
+        <h3 class="partner-h3">Partners</h3>
+        <ul class="parts">
+          <li>
+            <img class="partner-logo" src="./images/oil.png" alt="Tele" />
+          </li>
+          <li>
+            <img class="partner-logo" src="./images/work.png" alt="Work Oil" />
+          </li>
+          <li>
+            <img
+              class="partner-logo"
+              src="./images/Fibela-logo.png"
+              alt="Fibela Oil"
+            />
+          </li>
```

Comment:
- [ ] Please reduce the size of the images for the partners a little bit, right now they are too big and take up too much space as you can see below
> Your Design
![image](https://user-images.githubusercontent.com/90556221/181482002-566d7d18-1ada-4edc-8d6d-5a8c62b046e4.png)

> Behance
![image](https://user-images.githubusercontent.com/90556221/181482039-722ad2f0-a2ff-4c48-926f-b093cd1e5f80.png)

- [ ] Additionally, please also remove the default list styling for the layout. There are black dots visible on the screen which I have highlighted above